### PR TITLE
Add workflow: automatically publish docker image package when released

### PR DESCRIPTION
**Pull request by TianlongLiang**
#1607

### CODE REVIEW

File: .github/workflows/build_docker_images_package.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+on:
+  workflow_call:
+    inputs:
+      ver_num:
+        description: a semantic version number.
+        type: string
+        required: true
+
+jobs:
+  build-and-push-image:
```
Comment (wenyongh) [inline .github/workflows/build_docker_images_package.yml:15]: build-and-push-images?

File: .github/workflows/build_docker_images_package.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          string: ${{ github.actor }}
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ steps.downcase_github_actor.outputs.lowercase }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image(wasm-toolchain:1.0) to Container registry
```
Comment (NingW101) [inline .github/workflows/build_docker_images_package.yml:38]:
```suggestion
      - name: Build and push Docker image(wasm-toolchain:${{ inputs.ver_num }}) to Container registry
```

File: .github/workflows/build_docker_images_package.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        run: |
+          docker build -t ghcr.io/${{ steps.downcase_github_actor.outputs.lowercase }}/wasm-toolchain:${{ inputs.ver_num }} .
+          docker push ghcr.io/${{ steps.downcase_github_actor.outputs.lowercase }}/wasm-toolchain:${{ inputs.ver_num }}
+        working-directory: test-tools/wamr-ide/WASM-Toolchain/Docker
+
+      - name: Build and push Docker image(wasm-debug-server:1.0) to Container registry
```
Comment (NingW101) [inline .github/workflows/build_docker_images_package.yml:44]:
```suggestion
      - name: Build and push Docker image(wasm-debug-server:${{ inputs.ver_num }}) to Container registry
```

File: .github/workflows/build_docker_images.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Downcase github actor
+        id: downcase_github_actor
+        uses: ASzc/change-string-case-action@v2
+        with:
+          string: ${{ github.actor }}
+
+      - name: Log in to the Container registry
```
Comment (xujuntwt95329) [inline .github/workflows/build_docker_images.yml:31]:
```suggestion
      - name: Login to the Container registry
```

**Comment by wenyongh** [comment]
LGTM

**Pull request merged**